src/command/output.rs

```diff
@@ -262,6 +267,19 @@ impl RoverOutput {
 
                 stdoutln!("{}", skin.inline(explanation))?;
             }
+            RoverOutput::ReadmeFetchResponse(readme_response) => {
+                print_descriptor("Readme")?;
+                print_content(&readme_response)?;
+            }
+            RoverOutput::ReadmePublishResponse {
+                new_content,
+                last_updated_at,
+            } => {
+                print_descriptor("New Readme Content")?;
+                print_content(&new_content)?;
+                print_descriptor("Last Updated At")?;
+                print_content(&last_updated_at)?;
```

Comment:
we don't want the "last_updated_at" to be part of the actual `stdout` output here because it would end up getting piped to a file.
If I run the following I get this output to the terminal
```console
$ echo "this is my readme" | rover readme publish rover-journey --file - > command.output
Publishing graph variant README of rover-journey@current using credentials from the default profile.
```

and then the file `command.output` looks like this:

```
this is my readme
2022-05-23T17:29:16.193067000Z
```

which... isn't super useful or well structured.

for comparison, `rover graph publish` will only output the hash that it just published to. you can't currently use that hash anywhere, but the thinking is that eventually maybe you could fetch a graph by that hash, so consuming it with a machine would be helpful (and you wouldn't want any of the other informational human-style strings to get in there with it)

```console
$ echo "this is my sdl" | rover graph publish rover-journey --schema - > command.output
rover graph publish rover-journey --schema rover-journey.out > publish.out
Publishing SDL to rover-journey@current using credentials from the default profile.
rover-journey@current#4f06e6 Published successfully [No Changes]
---

and then the file `command.output` looks like this:

```
4f06e6
```

Honestly, I think I'd likely just do an `EmptySuccess` here as the response, and have a call to `stderrln` that lets us know that the publish was successful (make it as similar to `rover graph publish` as possible). I don't think we need to print the entire README to the terminal after publishing, they can always just fetch it if they want to see the whole thing.

Reply:
updated, now readme publish will show `{graphRef} Readme Published successfully` only. but the json output will still include the lastUpdatedAt timestamp. Is that correct?